geoh5py/shared/utils.py

```diff
@@ -262,57 +262,70 @@ def are_objects_similar(obj1, obj2, ignore):
     return attributes1 == attributes2
 
 
```

Comment:
**suggestion:** in `are_objects_similar()`, have type hints, and doc for param `ignore`:
```python
def are_objects_similar(obj1, obj2, ignore: list[str]) -> bool:
```

Reply:
done